src/components/Header/index.jsx

```diff
@@ -76,4 +83,8 @@ const Header = styled(HeaderComponent)`
 	box-shadow: 50px 150px 100px #232323 inset;
 `;
 
-export default Header;
+const mapDispatchToProps = (dispatch) => ({
+	createMovie: (data) => dispatch(createMovie(data)),
+});
+
+export default connect(null, mapDispatchToProps)(Header);
```

Comment:
You can also use another syntax without mapDispatchToProps
`export default connect(null, { createMovie })(Header);`